openscm/scmdataframe/base.py

```diff
@@ -575,46 +603,49 @@ def filter(
         Parameters
         ----------
         keep
-            If True, keep all timeseries satisfying the filters, otherwise drop all
-            the timeseries satisfying the filters
+            If True, keep all timeseries satisfying the filters, otherwise drop all the
+            timeseries satisfying the filters
 
         inplace
             If True, do operation inplace and return None
 
         has_nan
-            If True, convert all nan in ``meta_col`` to empty string before applying
-            filters. This means that "" and "*" will match rows with ``np.nan``. If
-            False, the conversion is not applied and so a search in a string column
-            which contains ``np.nan`` will result in a ``TypeError``.
+            If ``True``, convert all nan values in :obj:`meta_col` to empty string
+            before applying filters. This means that "" and "*" will match rows with
+            :class:`np.nan`. If ``False``, the conversion is not applied and so a search
+            in a string column which contains ;class:`np.nan` will result in a
+            :class:`TypeError`.
 
         **kwargs
             Argument names are keys with which to filter, values are used to do the
             filtering. Filtering can be done on:
 
-            - all metadata columns with strings, `*` can be used as a wildcard in
-              search strings
+            - all metadata columns with strings, "*" can be used as a wildcard in search
+              strings
 
-            - 'level': the maximum "depth" of IAM variables (number of
-              `self.data_hierarchy_separator`'s, excluding the strings given in the
-              'variable' argument)
+            - 'level': the maximum "depth" of IAM variables (number of hierarchy levels,
+              excluding the strings given in the 'variable' argument)
 
-            - 'time': takes a `datetime.datetime` or list of `datetime.datetime`'s
+            - 'time': takes a :class:`datetime.datetime` or list of
+              :class:`datetime.datetime`'s
+              TODO: default to np.datetime64
```

Comment:
is this still a TODO? DO we want to just do it here?

Reply:
I think we should default to datetime64in whole SCMDataFrame. Maybe a dedicated PR?

Reply:
ok make an issue then resolve this (I don't really understand what you mean by default to so can discuss in the issue)?

Reply:
see #183